apps/transformer/test/transform/add_timestamp_test.exs

```diff
@@ -0,0 +1,32 @@
+defmodule Transform.AddTimestampFieldTest do
+  use ExUnit.Case
+  use Placebo
+
```

Comment:
Should we write a test // handle the case where the transform tries to create a date field with the same name as a field which already exists?

Reply:
Sure. What should it do in that case?

Reply:
I dont know to be honest.  @jdenen ?

Reply:
I don't think we should write that test. I don't believe we're handling that case for the `MoveField` transformation either, and I'd rather we do it for both in another PR at the same time.

Reply:
I think we should not worry about that problem in the transformations themselves, that feels like something define should be catching.  When the user says I want a transformation, the dictionary can be checked for field collision then.  The more checks and conditions we keep out of the pipeline, the faster it will be.